CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # [unreleased]
 
+Breaking changes:
+
+* The list_room respone changes the fields `version`, `join_rules`, `guest_access` and `history_visibility` to be an option
+* The list_room respone changes the `join_rules` field to be `Option<SpaceRoomJoinRule>`
+
+Bug fixes:
```

Comment:
This is more of an improvement because the field was added after this endpoint was created.